var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -217,18 +264,17 @@ def install(self, spec, prefix):
                     cmakeOptions = []
                     cmakeOptions.extend([
                         '-DCMAKE_INSTALL_PREFIX=%s' % prefix,
-                        '-DCMAKE_LIBRARY_PATH=%s' % prefix.lib64,
-                        '-DINSTRUMENTOR=%s' % instrumentor_setting,
-                        '-DLIBMONITOR_DIR=%s' % spec['libmonitor'].prefix,
-                        '-DLIBUNWIND_DIR=%s' % spec['libunwind'].prefix,
-                        '-DPAPI_DIR=%s' % spec['papi'].prefix])
+                        '-DINSTRUMENTOR=%s'     % instrumentor_setting,
```

Comment:
i think you have quite a lot of code duplication here for `cmakeOptions` for all different combinations. I suggest you pull it out of all `if's` and set up what is common for all variants, and then `extend` it within each variant as needed.